#! /bin/sh
patch -p0 -f $* < $0
exit $?

2009-03-14 [colin]      3.7.1cvs12

        * src/common/utils.c
                Win32: fix subject prefix length

Index: src/common/utils.c
===================================================================
RCS file: /srv/cvs/claws-mail/claws/src/common/utils.c,v
retrieving revision 1.36.2.166
retrieving revision 1.36.2.167
diff -u -p -u -r1.36.2.166 -r1.36.2.167
--- src/common/utils.c	23 Feb 2009 16:55:48 -0000	1.36.2.166
+++ src/common/utils.c	14 Mar 2009 08:19:21 -0000	1.36.2.167
@@ -3773,8 +3773,12 @@ int subject_get_prefix_length(const gcha
 
 	for (n = 0; n < PREFIXES; n++) {
 		int len = strlen(prefixes[n]);
-		if (!strncasecmp(subject, prefixes[n], len))
-			return len;
+		if (!strncasecmp(subject, prefixes[n], len)) {
+			if (subject[len] == ' ')
+				return len+1;
+			else
+				return len;
+		}
 	}
 	return 0;
 #endif
